### ISSUE TITLE
AnyChart: Events handling not working

### ISSUE THREAD

**Issue by JelleDekker** [mendixlabs/charts#59]
When I define an Event entity, Event data attribute and On click microflow, nothing happens. The on click microflow is never triggered. Instead I see the following error:

`t is undefined, can't access property "eventEntity" of it r</l</t.prototype.onClick@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/AnyChart.js?636736545135115089:17:55679
t/t.onClick@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/AnyChart.js?636736545135115089:17:2373
n/<[194]</n.prototype.emit@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:398178
init/t.emit@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:1864029
o@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:1720692
n/<[510]</e.exports@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:1720817
clickFn@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:2020773
A@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:1691000`

This error is also shown when Event entity, Event data attribute and On click microflow are empty.

**Comment by djkwagala** [comment]
Hi Jelle,
Thanks for reporting this issue, we've added it to the backlog. Once the solution is ready, you will be notified immediately

Regards
Derrick Kwagala

**Comment by JelleDekker** [comment]
Found another issue with event handling, the tooltips are also not working
![image](https://user-images.githubusercontent.com/1491873/46293377-98401100-c593-11e8-9ccf-a5ab7720bc9f.png)

**Comment by djkwagala** [comment]
@JelleDekker 
Could you please show me your modeler configuration. With correct configuration, I'm unable to reproduce this error.

**Comment by djkwagala** [comment]
> When I define an Event entity, Event data attribute and On click microflow, nothing happens. The on click microflow is never triggered. Instead I see the following error:
> 
> `t is undefined, can't access property "eventEntity" of it r</l</t.prototype.onClick@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/AnyChart.js?636736545135115089:17:55679 t/t.onClick@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/AnyChart.js?636736545135115089:17:2373 n/<[194]</n.prototype.emit@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:398178 init/t.emit@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:1864029 o@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:1720692 n/<[510]</e.exports@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:1720817 clickFn@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:2020773 A@http://localhost:8080/widgets/com/mendix/widget/custom/AnyChart/chunke87d7e2b1e7e44ce8a15.js:1:1691000`
> 
> This error is also shown when Event entity, Event data attribute and On click microflow are empty.

We are currently unable to reproduce this issue.

**Comment by JelleDekker** [comment]
I've attached a zip with a project package, it shows both bugs I'm describing. It's a blank app in Mx 7.18.1 with the latest version of AnyCharts widget from the appstore.
[AnyChartEvents.zip](https://github.com/mendixlabs/charts/files/2441980/AnyChartEvents.zip)
